docs/extend/console.md

```diff
@@ -39,8 +39,23 @@ return [
 ];
 ```
 
-::: tip Scheduled Commands
+## Scheduled Commands
 
-The [fof/console library](https://github.com/FriendsOfFlarum/console) allows you to schedule commands to run on a regular interval! However, please note that this is a community solution.
+The `Flarum\Extend\Console`'s `schedule` method allows extension developers to create scheduled commands that run on an interval:
 
-:::
+
+```php
+use Flarum\Extend;
+use YourNamespace\Console\CustomCommand;
+use Illuminate\Console\Scheduling\Event;
+
+return [
+  // Other extenders
```

Comment:
Don't we format PHP examples with 4 spaces?

Reply:
I think we need something like https://github.com/rposborne/prettier-markdown-codeblocks